### PR TITLE
Bump releases

### PR DESCRIPTION
**Pull request by irbekrm**
This PR:

- bumps cert-manager release tags as we just released v1.5
- run presubmits against Kubernetes v1.22 by default (before it was v1.21)
- adds a periodic for release next to run tests on Kubernetes v1.22

Signed-off-by: irbekrm <irbekrm@gmail.com>

### CODE REVIEW

**Review by wallrj — APPROVED** [review]
There was a bit more to the PR last you did this: https://github.com/jetstack/testing/pull/515/files

but I think that was because we also added K8S 1.21 testing at the same time.

/approve
/lgtm

**Review by SgtCoDFish — COMMENTED** [review]
Do we want to add 1.22 testing for all PRs against master in this change too? I think it'd be valuable; feel free to unhold if we should do that in a different PR! (EDIT: or if we don't want to do that :grin: )

(If we don't change it here, we'll need to change the supported releases page, which I'll keep an eye on)

/hold

**Comment by irbekrm** [comment]
> Do we want to add 1.22 testing for all PRs against master in this change too? I think it'd be valuable; feel free to unhold if we should do that in a different PR! (EDIT: or if we don't want to do that grin )

I wanted to get https://github.com/jetstack/cert-manager/pull/4341 merged first

File: config/jobs/cert-manager/cert-manager-periodics.yaml
Comment (irbekrm) [inline config/jobs/cert-manager/cert-manager-periodics.yaml:438]: This is not a new job. I swapped some job definitions in the file and git displays the change in a weird way.

**Comment by irbekrm** [comment]
> Do we want to add 1.22 testing for all PRs against master in this change too? 

Thanks @SgtCoDFish I've updated PR.

**Review by SgtCoDFish — APPROVED** [review]
/lgtm

I think there'd be some value to us running both v1.21 and v1.22 e2e tests on every PR, but I don't think it's _required_ and given the flakiness we've seen from our tests, it might well do more harm than good to test against 2 versions every time :sweat_smile: 

After this is merged, I'll update [this page](https://github.com/cert-manager/website/blob/master/content/en/docs/installation/supported-releases.md#how-we-determine-supported-kubernetes-versions-kubernetes-supported-versions) to reflect exactly which versions we test against.

**Comment by irbekrm** [comment]
> I think there'd be some value to us running both v1.21 and v1.22 e2e tests on every PR, but I don't think it's required 

Yeah I was thinking about that too. Not strongly convinced either way, but we are also resource limited for test nodes 😿 

Maybe I will just unhold this, running both if we decide so would be just a matter of changing two lines, shout if you disagree

/hold cancel

**Comment by irbekrm** [comment]
FWIW I don't understand why the PRs against this repo don't get automatically merged.
As I understand it is Prow's tide component that should merge this and looking at its config it seems to be that [this is configured against this repo correctly](https://github.com/jetstack/testing/blob/master/config/config.yaml#L264-L282).

**Comment by irbekrm** [comment]
Seems like some unhappiness in Prow:

```
{"base-sha":"7c81219cd37d6248ee47f0f07856bd1700feb49a","branch":"master","component":"tide","controller":"sync","error":"status code 422 not one of [201], body: {\"message\":\"Validation Failed\",\"errors\":[{\"resource\":\"Status\",\"code\":\"custom\",\"message\":\"This SHA and context has reached the maximum number of statuses.\"}],\"documentation_url\":\"https://docs.github.com/rest/reference/repos#create-a-commit-status\"}","file":"prow/tide/tide.go:1197","func":"k8s.io/test-infra/prow/tide.(*Controller).mergePRs","level":"error","merge-targets":[528,529],"msg":"Unable to set tide context to SUCCESS.","org":"jetstack","pr":529,"repo":"testing","severity":"error","sha":"8fbbfe5e94f828ffcb5a1bda400ad2132be239f0","time":"2021-08-12T14:11:42Z"}
{"action":"MERGE_BATCH","base-sha":"7c81219cd37d6248ee47f0f07856bd1700feb49a","branch":"master","component":"tide","controller":"sync","file":"prow/tide/tide.go:1625","func":"k8s.io/test-infra/prow/tide.(*Controller).syncSubpool","level":"info","msg":"Subpool synced.","org":"jetstack","repo":"testing","severity":"info","targets":[528,529],"time":"2021-08-12T14:11:42Z"}
{"base-sha":"7c81219cd37d6248ee47f0f07856bd1700feb49a","branch":"master","component":"tide","controller":"sync","error":"failed merging [528 529] from batch [528 529]: status code 422 not one of [201], body: {\"message\":\"Validation Failed\",\"errors\":[{\"resource\":\"Status\",\"code\":\"custom\",\"message\":\"This SHA and context has reached the maximum number of statuses.\"}],\"documentation_url\":\"https://docs.github.com/rest/reference/repos#create-a-commit-status\"}","file":"prow/tide/tide.go:425","func":"k8s.io/test-infra/prow/tide.(*Controller).Sync.func2","level":"error","msg":"Error syncing subpool.","org":"jetstack","repo":"testing","severity":"error","time":"2021-08-12T14:11:42Z"}
```

I see that a similar issue [has been fixed a while ago](https://github.com/kubernetes/test-infra/pull/15300) (and we are running a tide version with that fix). Perhaps something to keep in mind and investigate in case PR merging breaks for cert-manager as well.

**Pull request merged**
